code/src/album.css

```diff
@@ -0,0 +1,122 @@
+
+:root {
+  --primary-headings-font-family: "Helvetica", sans serif;
+  --secondary-headings-font-family: "Helvetica", sans serif;
+  --paragraph-font-family: "Helvetica", sans serif;
+}
+
+/*ALBUM*/
+
+.album-name {
+  font-family: var(--secondary-headings-font-family);
+  font-size: 14px;
+  color: #fff;
+  text-decoration: none;
+}
+
+.album {
+  margin-bottom: 1.5rem;
+}
+
+.album-name:hover {
+  text-decoration: underline;
+}
+
+.album-cover {
+  width: 100%;
+  height: 100%;
+}
+
+/*album-cover gets opacity at hover on its parent element cover-and-icons*/
+
+.cover-and-icons:hover .album-cover {
+  opacity: 0.5;
+  cursor: pointer;
+}
+
+.album-container {
+  place-items: center;
```

Comment:
This is a new one for us - nice to see and we'll probably use this in the future ourselves. We fixed it a different way.
/Ylva & Emma